docs/docs-content/clusters/edge/upgrade-behavior.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label: "Edge Cluster Upgrade Behavior"
+title: "Edge Cluster Upgrade Behavior"
+description: "Learn about how Palette Edge responds to upgrades of the cluster profile."
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge", "architecture"]
+---
+
+When you update an active Edge cluster's profile, Palette will upgrade the active cluster to the latest version of the
+profile. Depending on the nature of the change, implementing an upgrade might involve repaving a cluster, rebooting a
+cluster, restarting services, or doing nothing. For more information about cluster repaves, refer to
+[Repave Behavior and Configurations](../cluster-management/node-pool.md#repave-behavior-and-configuration).
```

Comment:
@lennessyy can we link to this page in `/clusters/cluster-management/cluster-updates` - I think that would help with cross reference.